### DISCUSSION TITLE
feat: Multiple Model Paths

### DISCUSSION THREAD

**Discussion by AdaptiveStep** [open-webui/open-webui#2330] [General]
**Is your feature request related to a problem? Please describe.**
There can only be one model path to ollama right now. And if ollama is installed naively on the native os it will use the immediate default path. The models can easily sum up to hundreds of GB. Additional model paths are required due to limitations in high speed nvme storage drives.

**Describe the solution you'd like**
Add additional paths for ollama to find its models, and a default folder, where all the models are put when pulling them down from the online library. This would let people use multiple drives for ollama. 

**Describe alternatives you've considered**
The alternative is to delete them and and keep pulling them down from the web when necessary. This will just cause unnecessary traffic to the ollama model host. 

**Additional context**
No screenshots

**Comment by justinh-rahb** [comment]
This is a limitation of Ollama. See if there's an issue open already for it, or go create one there.

**Reply by ZerocoolZa** [reply]
agreed even  with a 4tb nvme i have not enough space to. so mklink /D "C:\Users\Administrator\.ollama\combined_models" "C:\Users\Administrator\.ollama\models
" 

need more features in open web-- i made a exe - to check the open web ui - download the git -  and run locally -- Beter than a docker container- maximizing Resources --
